app/src/main/java/mozilla/lockbox/presenter/SettingPresenter.kt

```diff
@@ -158,7 +160,7 @@ class SettingPresenter(
                 toggleObserver = sendUsageDataObserver
             ),
             AppVersionSettingConfiguration(
-                text = "App Version: $versionNumber",
+                text = "App Version: $versionName ($versionNumber)",
```

Comment:
Can we put this string into a resources?

Reply:
I think I'd prefer to put the`@StringRes Int` and the version name and number in to the `AppVersionSettingConfiguration` and then combine it in a `setText` method in [`AppVersionSettingViewHolder`][1]

[1]: https://github.com/mozilla-lockbox/lockbox-android/blob/b1ee56c4949d36762c554d84aca259de13de6aaf/app/src/main/java/mozilla/lockbox/view/SettingViewHolder.kt#L104